doc/releases/changelog-dev.md

```diff
@@ -225,67 +227,80 @@
   0.017750012490703237
   ```
 
-* New PennyLane-inspired `sketch` and `sketch_dark` styles are now available for drawing circuit diagram graphics.
-  [(#2709)](https://github.com/PennyLaneAI/pennylane/pull/2709)
-
-* Added `QutritDevice` as an abstract base class for qutrit devices.
-  ([#2781](https://github.com/PennyLaneAI/pennylane/pull/2781), [#2782](https://github.com/PennyLaneAI/pennylane/pull/2782))
-
-* Added operation `qml.QutritUnitary` for applying user-specified unitary operations on qutrit devices.
-  [(#2699)](https://github.com/PennyLaneAI/pennylane/pull/2699)
+  - Relative entropy support in `qml.math` for flexible post-processing:
 
-**Operator Arithmetic:**
+  ```pycon
+  >>> rho = np.array([[0.3, 0], [0, 0.7]])
+  >>> sigma = np.array([[0.5, 0], [0, 0.5]])
+  >>> qml.math.relative_entropy(rho, sigma)
+  tensor(0.08228288, requires_grad=True)
+  ```
 
-* Adds the `Controlled` symbolic operator to represent a controlled version of any
-  operation.
-  [(#2634)](https://github.com/PennyLaneAI/pennylane/pull/2634)
+* New PennyLane-inspired `sketch` and `sketch_dark` styles are now available for 
+  drawing circuit diagram graphics.
+  [(#2709)](https://github.com/PennyLaneAI/pennylane/pull/2709)
 
-* Adds a base class `qml.ops.op_math.SymbolicOp` for single-operator symbolic
-  operators such as `Adjoint` and `Pow`.
-  [(#2721)](https://github.com/PennyLaneAI/pennylane/pull/2721)
+<h4>Quality-of-life upgrades to Operator arithmetic 📈</h4>
```

Comment:
There seems to be a lot of extraneous information in this section of the changelog. Wondering if I can get a hand here to distill out what is and isn't extraneous?

Reply:
Yes I will give a look at it but we will see many PRs merged in the upcoming days 👍